packages/next/src/server/lib/parse-stack.ts

```diff
@@ -36,19 +31,6 @@ export function parseStack(
 
   const frames = parse(stack)
   return frames.map((frame) => {
-    try {
-      const url = new URL(frame.file!)
-      const res = regexNextStatic.exec(url.pathname)
-      if (res) {
-        const effectiveDistDir = distDir
-          ?.replace(/\\/g, '/')
-          ?.replace(/\/$/, '')
-        if (effectiveDistDir) {
-          frame.file =
-            'file://' + effectiveDistDir.concat(res.pop()!) + url.search
-        }
-      }
-    } catch {}
```

Comment:
This passed a different source URL to the server which is now asked to manually extract the sourcemapping URL from the file. We no longer need to do this.

We can probably remove this entire implementation. I just removed this because it's blocking https://github.com/vercel/next.js/pull/81509